ext/pdo_sqlite/tests/bug_44159_sqlite_version.phpt

```diff
@@ -0,0 +1,27 @@
+--TEST--
+PDO Common: Bug #44159 (Crash: $pdo->setAttribute(PDO::STATEMENT_ATTR_CLASS, NULL)): SQLite variant
```

Comment:
Test title seems odd?

Reply:
Well it's a copy paste from the original with an SQLite variant just appended

Reply:
Okay, what confused me was the reference to STATEMENT_ATTR_CLASS in the title, even though it's not used in the test.

Reply:
Ah right, I'll drop that then